config.yaml

```diff
@@ -21,7 +21,10 @@ external_services:
   # Uncomment istio_identity_domain to set a different value. This value must match the Istio configuration.
   # Default is "svc.cluster.local" (which matches Istio's default)
   # istio_identity_domain: svc.cluster.local
-
+  jaeger:
+    url: https://jaeger-query-istio-system.127.0.0.1.nip.io
```

Comment:
Same here, developers installation are not usually deployed under "https" scheme, I think.